domains/nabeeltahir.json

```diff
@@ -0,0 +1,11 @@
+
+{
+    "owner": {
+        "username": "nabeeltahir",
+        "email": "",
+        "discord": "934011704378032128"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=nabeeltahir.is-a.dev,e30f1de333781749cc70"
```

Comment:
This should go under a file called _vercel.domain-you-requested.json

See docs: https://docs.is-a.dev